### PR TITLE
changed the core version to include x.x.x

### PR DESCRIPTION
**Pull request by piyush0810**
Made changes for tagging the docker image for supertoken core postgres to enable pushing of x.x.x along with x.x . 

Tests performed:
Added multi tagging and fetching the version from docker file. Fetching works fine, tried in local shell and for the tags checked for a dummy container using same syntax, working fine. For multi tags:
Ran following command locally and tested both tags are present on dockerhub. 
`docker buildx build -t piyush0810/temp:1 -t piyush0810/temp:2 -o type=image,push=true .`

### CODE REVIEW

**Review by bhumilsarvaiya — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
       - run:
           name: core version tag
           command: |
-            CORE_VERSION=$(cat Dockerfile | grep "ARG CORE_VERSION=" | cut -d'=' -f2 | cut -d'.' -f1,2)
+            CORE_VERSION=$(cat Dockerfile | grep "ARG CORE_VERSION=" | cut -d'=' -f2)
```
[the remark this answers is not in the document]
Comment (piyush0810) [reply .circleci/config.yml]: reverted the change,

File: .circleci/config.yml [hunk 2]
```diff
@@ -37,8 +37,13 @@ jobs:
             docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
             docker buildx create --name multiarch --driver docker-container --use
             docker buildx inspect --bootstrap
+            CORE_VERSION_WITH_PATCH=$(cat Dockerfile | grep "ARG CORE_VERSION=" | cut -d'=' -f2)
             CORE_VERSION=$(cat Dockerfile | grep "ARG CORE_VERSION=" | cut -d'=' -f2 | cut -d'.' -f1,2)
-            docker buildx build --platform $BUILDX_PLATFORMS -t supertokens/supertokens-postgresql:${CORE_VERSION} -o type=image,push=true .
+            if [ "$CORE_VERSION" != "$CORE_VERSION_WITH_PATCH" ]; then
```
Comment (bhumilsarvaiya) [inline .circleci/config.yml:42]: no need have this if condition. Here, CORE_VERSION will always be X.Y and CORE_VERSION_WITH_PATCH will always be X.Y.Z. Which means the condition `if [ "$CORE_VERSION" != "$CORE_VERSION_WITH_PATCH" ]` can never be false.

File: .circleci/config.yml [hunk 3]
```diff
@@ -37,8 +37,13 @@ jobs:
...
+              docker buildx build --platform $BUILDX_PLATFORMS -t supertokens/supertokens-postgresql:${CORE_VERSION} -t supertokens/supertokens-postgresql:${CORE_VERSION_WITH_PATCH} -o type=image,push=true .
```
Comment (bhumilsarvaiya) [inline .circleci/config.yml:43]: Have you tested that all the tags will get pushed to docker hub while building the image this way? If not, you can create a docker hub account and try creating a random image with multiple tags as how you have done here and push. Verify that all the tags are present for the image.

**Review by bhumilsarvaiya — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (piyush0810) [reply .circleci/config.yml:43]: Tested, works fine.

File: .circleci/config.yml (hunk 2 above)
Comment (piyush0810) [reply .circleci/config.yml:42]: I wrote this condition only for the case when n docker file we ignore patch, in that case these will be same. Assuming that never happens, ignoring it for now.

**Review by bhumilsarvaiya — APPROVED** [review]

**Pull request merged**